public/components/FeatureAnywhereContextMenu/Container/Container.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import CreateAnomalyDetector from '../CreateAnomalyDetector';
+import { useIndex } from '../../../utils/contextMenu/indexes';
+import './styles.scss';
+
+const Container = ({ startingFlyout, ...props }) => {
```

Comment:
What is the purpose of this container exactly? Recommend naming it to something more descriptive.

Reply:
We use this to decide the flip between create checkable  card and associate existing checkable card. Will rename it to `DetectorContainer`